packages/twenty-server/src/engine/core-modules/workspace-invitation/services/workspace-invitation.service.ts

```diff
@@ -133,7 +137,24 @@ export class WorkspaceInvitationService {
     );
   }
 
-  // QUERY METHODS
+  async findInvitationByWorkspaceSubdomainAndUserEmail({
+    subdomain,
```

Comment:
Overall, I feel that subdomain / domain logic should stay in DomainManager module and that we should keep passing workspaceId in the rest of the code